src/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+find_package(Freetype REQUIRED)
```

Comment:
We don't define `project` or other items as these aren't meant to be built outside the root cmake file